packages/editor/src/client/panel/peerEditPanel.tsx

```diff
@@ -30,12 +30,13 @@ import {RichTextBlock} from '../blocks/richTextBlock/richTextBlock'
 
 export interface ImageEditPanelProps {
   id?: string
+  hostURL?: string
```

Comment:
don't make this optional. You should always have a `hostURL`

Reply:
When there's no id?